AsyncDisplayKit/Details/Transactions/_ASAsyncTransaction.mm

```diff
@@ -508,7 +508,7 @@ - (void)_ensureTransactionData
 
 - (NSString *)description
 {
-  return [NSString stringWithFormat:@"<_ASAsyncTransaction: %p - _state = %lu, _group = %p, _operations = %@>", self, (unsigned long)_state, _group, _operations];
+  return [NSString stringWithFormat:@"<_ASAsyncTransaction: %p - _state = %lu, _group = %p, _operations = %@>", self, (unsigned long)__atomic_load_n(&_state, __ATOMIC_SEQ_CST), _group, _operations];
```

Comment:
Why is this not an issue for accessing other instance variables / shared state outside of a locked / critical section?  E.g. _group, _operations.